src/tests/e2e/wishlist.test.js

```diff
@@ -96,7 +96,7 @@ describe('Wishlist', () => {
   test('Remove wishlist item', async () => {
     const browser = await puppeteer.launch({
       args: ['--no-sandbox'],
-      slowMo: 30,
+      slowMo: 120,
```

Comment:
This might run successfully locally, but would timeout in Jenkins since the total test timeout is relatively slow.